contracts/tokens/BaseToken.sol

```diff
@@ -153,5 +164,32 @@ abstract contract BaseToken is Context, ERC721Enumerable, CommunityOwnable {
         super._beforeTokenTransfer(from, to, firstTokenId, batchSize);
     }
 
+    /**
+     * @dev See {IERC1155-safeBatchTransferFrom}.
+     */
```

Comment:
This natspec needs to change as our `safeBatchTransfer()` implementation isn't actually `ERC1155` compliant.